Vector/AppDelegate.m

```diff
@@ -426,13 +418,6 @@ - (void)applicationWillTerminate:(UIApplication *)application
 
 #pragma mark - Application layout handling
 
-- (void)showAuthenticationScreen
-{
-    [self restoreInitialDisplay:^{
-        [_homeViewController performSegueWithIdentifier:@"showAuth" sender:self];
-    }];
-}
-
 - (void)popRoomViewControllerAnimated:(BOOL)animated
```

Comment:
May worth to change the name of this method because it does not correspond to what the function actually does.
Proposal: popToHomeViewControllerAnimated
It does not need to be public.